drivers/regex/mlx5/mlx5_regex_fastpath.c

```diff
@@ -257,7 +257,7 @@ mlx5_regexdev_dequeue(struct rte_regexdev *dev, uint16_t qp_id,
 
 	while ((cqe = poll_one(cq))) {
 		uint16_t wq_counter
-			= (be16toh(cqe->wqe_counter) + 1)%MAX_WQE_INDEX;
+			= (rte_be_to_cpu_16(cqe->wqe_counter) + 1)%MAX_WQE_INDEX;
```

Comment:
same as above, changes should be on relevant patch

Reply:
do you think it will be better to have index & (MAX_WQE_INDEX-1) or the modulo?
this is for other indexes also.

Reply:
Definitely modulu